### DISCUSSION TITLE
Telnet to Com Port

### DISCUSSION THREAD

**Discussion by meluvalli** [86Box/86Box#3477] [Ideas]
Hello.  In DOSBox, there is an option to have a port open to telnet into a virtual comport of the Virtual Machine.  Is this possible to add something like this?

Example, if you local internal host IP is 192.168.1.100, you could assign COM Port 1 to port 5033 for example, and telnet into 192.168.1.100 port 5033 and it would go into the virtual machine as an incoming call for a modem.

Would be nice to see this for using with BBS systems.

**Comment by cold-brewed** [comment]
This doesn't exist natively in 86Box but you could probably accomplish the same thing with `socat` and serial passthrough.

**Comment by www2000** [comment]
I like to have this feature in 86box.
I look in some old software that use a comport but i wand to pass the data to my own software using a TCP sockets than over my pc com interface.
